packages/application/test/shell.spec.ts

```diff
@@ -517,4 +517,61 @@ describe('LabShell', () => {
       expect(widget.isVisible).toBe(false);
     });
   });
+
+  describe('#ariaLabelsAndRoles', () => {
```

Comment:
I would name the group `accessibility` in case if we want to add more tests which do not fit neatly to specific methods or attributes in the future.

Reply:
Changes made.